[Q3 2025] docs: adjust articles for the upcoming ai-related endpoint changes (#1775)
telerik/reporting-docs: embedding-reports/host-the-report-engine-remotely/rest-api-reference/features-api/create-ai-thread.md

---
title: Create AI Thread
page_title: Create AI Thread Request Explained
description: "Learn how to make the Create AI Thread request to the Telerik Reporting REST Service and what response to expect."
slug: telerikreporting/using-reports-in-applications/host-the-report-engine-remotely/telerik-reporting-rest-services/rest-api-reference/features-api/create-ai-thread
tags: create,ai,thread
published: True
position: 1
---

# `Create AI Thread` Overview

The `Create AI Thread` endpoint initializes a new AI interaction session for the Telerik Reporting REST Service. It provides a unique thread identifier, available AI prompts, and consent requirements.

## Request

	POST /api/reports/clients/{clientId}/instances/{instanceId}/ai

__Path parameters__

| Name | Type | Description |
| ------ | ------ | ------ |

> |`clientId`|String|ID of the service client the AI thread is started in. Returned from [Register Client]({%slug telerikreporting/using-reports-in-applications/host-the-report-engine-remotely/telerik-reporting-rest-services/rest-api-reference/clients-api/register-client%}).|

Reviewer: ```suggestion
|`clientId`|String|The ID of the service client that started the AI thread. Returned by [Register Client]({%slug telerikreporting/using-reports-in-applications/host-the-report-engine-remotely/telerik-reporting-rest-services/rest-api-reference/clients-api/register-client%}).|
```

> |`instanceId`|String|ID of the report instance used for the AI thread. Returned from [Resolve Report Instance]({%slug telerikreporting/using-reports-in-applications/host-the-report-engine-remotely/telerik-reporting-rest-services/rest-api-reference/report-instances-api/resolve-report-instance%}).|

Reviewer: ```suggestion
|`instanceId`|String|The ID of the report instance used for the AI thread. Returned by [Resolve Report Instance]({%slug telerikreporting/using-reports-in-applications/host-the-report-engine-remotely/telerik-reporting-rest-services/rest-api-reference/report-instances-api/resolve-report-instance%}).|
```
